tock/hours/views.py

```diff
@@ -816,10 +820,11 @@ def last_timecard(self):
     def prefilled_formset(self):
         timecard = self.last_timecard()
         timecard_prefills = dict(
-            TimecardPrefillData.objects.active().filter(
-                employee=self.request.user.user_data).values_list(
-                    'project', 'hours'
-                )
+            TimecardPrefillData
+            .objects
+            .active()
+            .filter(employee=self.request.user.user_data)
+            .values_list('project', 'hours')
         )
```

Comment:
I think that was way more readable before, especially given the nested queryset.  I'd probably get the nested list first, then pass it in. Also, maybe the `active()` manager could wrap `objects` and tighten things up too. Maybe.